pkg/snapshot/README.md

```diff
@@ -42,13 +42,18 @@ go build
 ## Preparation
 1. Use API Category to group the APIs you want to export
 
-By default, snapshot tool will export all APIs from Tyk dashboard with category `operator`.  You can also categorize your APIs by teams, environments, or any way you want. You can specify which API category to be exported in later steps.
+By default, snapshot tool will export all APIs from Tyk dashboard with category `operator`.  
+You can also categorize your APIs by teams, environments, or any way you want. 
+You can specify which API category to be exported in later steps.
```

Comment:
Why by default will operator only pull categorised `operator`? Default should be ALL apis, and perhaps optionally filter by category?

Reply:
>Default should be ALL apis, and perhaps optionally filter by category?

Yes, exactly! Updated accordingly